radar/2024-12-01/event-sourcing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Event Sourcing"
```

Comment:
Ah.. good old event sourcing. It has been on our radar before I'm pretty sure (years ago). If we really want to properly trial it this time we should really plan this. Do we have a plan for this already? In my experience it has typically proven overkill for the solutions we built and hence it has fallen off our radar time and again. But now that we are moving more towards data science and data platforms it could be interesting again to look at it I think..

Reply:
Yes, with doing more and more data platforms and IoT devices as well, I think we need to use this for some of the upcoming projects, I would plan to use it at least partly for JII